DBlock Game/DBlock Game/Program.cs

```diff
@@ -1347,6 +1355,133 @@ public static void Fight()
             Menu();
         }
 
+        public static void Death()
+        {
+            Random rand = new Random();
+            string player;
+            string comp;
+            int pscore = 0;
+            int cscore = 0;
+
+            // the paper,scissors,rock game
+
+            for (int i = 0; i < 4; i++)
+            {
+                player = "";
```

Comment:
if u wanted to get rid of the assigning lines you could have a do while loop instead of the while loop and you wouldnt need to assign player or comp anything at the start.